test/integration/default/serverspec/instrumental_tools_spec.rb

```diff
@@ -1,30 +1,57 @@
 require 'serverspec'
 
-set :backend, :exec
+details = {}
+
+if RUBY_PLATFORM =~ /(win|mswin|mingw)/i
+  set :backend, :cmd
+  set :os, :family => 'windows'
+  details = {
+    check_executable: false,
+    check_owner:      false,
+    config:           "c:\\Program Files (x86)\\Instrumental Tools\\etc\\instrumental.yml",
```

Comment:
Agree, I didn't find an obvious way to pull this from the Chef attributes of the recipe that this is testing, so just deferred to hardcoding it since we control all the defaults in these tests anyways.